src/extension/monitor/core/src/main/java/org/geoserver/monitor/Monitor.java

```diff
@@ -73,7 +73,7 @@ public RequestData start() {
 
         // notify listeners
         for (RequestDataListener listener : listeners) {
-            listener.requestUpdated(req);
+            listener.requestStarted(req);
```

Comment:
The 'requestStarted' method doesn't seem to be used anywhere else.  I'm recommending this change since the data provided this early in the request seems somewhat limited. 

This change would allow only those who want to see data at the very beginning of the MonitorFilter to get access to it.

Reply:
The RequestDataListener interface provides basic lifecycle hooks, similar to DispatcherCallback. Some are more informative than others, the reason why they are there is to allow following a request cycle. 
I don't understand why you'd want to break an existing extension point (not necessarily used only inside GeoServer), is there a significant overhead in this call? Please elaborate.

Reply:
The overhead isn't necessarily significant (since that'd depend on the implementation).

I am asking about this since I was surprised by the number of calls to requestUpdated.  After I put the PR, I realized that as an implementer, I can ask some quick checks to skip over some of the calls which provide less information than I'd care to report/record.